test/nuts/mdapi.nut.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from 'chai';
+
+import { TestSession, execCmd } from '@salesforce/cli-plugins-testkit';
+
+let session: TestSession;
+
+describe('mdapi NUTs', () => {
+  before(async () => {
+    session = await TestSession.create({
+      project: {
+        gitClone: 'https://github.com/trailheadapps/dreamhouse-lwc.git',
+      },
+      setupCommands: [
+        // default org
+        'sfdx force:org:create -d 1 -s -f config/project-scratch-def.json',
+      ],
+    });
+    process.env.SFDX_USE_PROGRESS_BAR = 'false';
+  });
+
+  after(async () => {
+    await session?.zip(undefined, 'artifacts');
+    await session?.clean();
+  });
+
+  describe('mdapi:listmetadata', () => {
+    it('should successfully execute listmetadata', () => {
+      const result = execCmd('force:mdapi:listmetadata --json --metadatatype CustomObject');
+      expect(result.jsonOutput.status).to.equal(0);
+      expect(result.jsonOutput.result).to.be.an('array').with.length.greaterThan(100);
+      expect(result.jsonOutput.result[0]).to.have.property('type', 'CustomObject');
+    });
+  });
+
+  // describe('Test stash', () => {
```

Comment:
do we need a failure NUT? maybe a type that doesn't exist in the org

also can remove these commented out NUTs

Reply:
There isn't a failure NUT for list because it's covered in the unit tests.  I don't want to uncomment or remove the other tests because they came from toolbelt and I want to uncomment them as things are moved over, or at least know what the tests used to be.